Automattic/wp-calypso
# Adding Whats New modal to the block editor

Adding a new option to the edit post menu in Gutenberg for WPcom (pau2Xa-2eS-p2 for context).

This PR adds a What's New Guide in the WPcom block editor that highlights a predefined list of new features.

The guide is accessible via the sidebar menu circled in this screenshot:
<img width="1516" alt="Screen Shot 2021-01-21 at 10 08 28 AM" src="https://user-images.githubusercontent.com/35781181/105371307-7a6aa000-5bd2-11eb-874c-ed6a8177930a.png">

Clicking that icon pops the sidebar menu. You'll see a new "What's New" option in the "Tools" submenu. Clicking that link pops the What's New guide, which looks like this:
<img width="1200" alt="Screen Shot 2021-01-21 at 10 11 57 AM" src="https://user-images.githubusercontent.com/35781181/105371447-9ff7a980-5bd2-11eb-8412-de26beeb1569.png">

#### Testing instructions

* Checkout this PR and sync the editor toolkit plugin to your sandbox.
* Sandbox one of your test sites.
* Load the post editor and navigate the menus as shown above.
* Load the What's New guide, read the cards, and click the links to confirm they work as expected.

Fixes #
